crates/aptos/src/governance/mod.rs

```diff
@@ -279,7 +279,7 @@ pub struct SubmitProposal {
     #[clap(long)]
     pub(crate) metadata_path: Option<PathBuf>,
 
-    #[clap(long, default_value = "false")]
```

Comment:
Does this still somehow default to true?

Reply:
Yep.

Reply:
That's so unfortunate